RecoEgamma/EgammaElectronAlgos/src/RegressionHelper.cc

```diff
@@ -193,6 +193,7 @@ void RegressionHelper::applyCombinationRegression(reco::GsfElectron & ele) const
     break;
   default:
     elClass = -1;
+  };
```

Comment:
@slava77 The change is coming from this line, which is a legitimate fix. I'll remove it from this PR and start another one with only this fix.